app/Main.hs

```diff
@@ -11,8 +11,11 @@ import Polysemy
 import Servant.Client (BaseUrl(..), Scheme(Http))
```

Comment:
Some of these imports are redundant.
Please run the pipeline against your PR.

Reply:
Done